### PR TITLE
Initial server quick start guide docs added.

### PR DESCRIPTION
**Pull request by kasenvr**

### CODE REVIEW

File: docs/source/host/quick-start.rst
Comment (ctrlaltdavid) [inline docs/source/host/quick-start.rst:12]:
```suggestion
There are two ways to make a domain accessible from the outside Internet.
```

File: docs/source/host/quick-start.rst
Comment (ctrlaltdavid) [inline docs/source/host/quick-start.rst:24]:
```suggestion
1. Download Vircadia (full, not Interface only).
```

File: docs/source/host/quick-start.rst
Comment (ctrlaltdavid) [inline docs/source/host/quick-start.rst:30]:
```suggestion
4. Go to ``http://localhost:40100/`` in your web browser and complete the wizard to set up your domain. If the server is on a different machine, then replace 'localhost' with its IP address.
```

File: docs/source/host/quick-start.rst
Comment (ctrlaltdavid) [inline docs/source/host/quick-start.rst:31]:
```suggestion
5. In your Vircadia Interface, open the `Explore` app and enter `localhost` (or the server's IP address) to get there.
```

File: docs/source/host/quick-start.rst
Comment (ctrlaltdavid) [inline docs/source/host/quick-start.rst:25]:
```suggestion
2. Use the `Express` install option, or if you use the `Custom` option ensure that both the `Vircadia Interface` and `Vircadia Sandbox` options are checked. Vircadia Sandbox is the domain server.
```

**Review by ctrlaltdavid — CHANGES_REQUESTED** [review]

File: docs/source/host/quick-start.rst
Comment (kasenvr) [reply docs/source/host/quick-start.rst:12]: This is a bit of an odd one, personally I wouldn't capitalize it, I don't capitalize "metaverse" either typically but apparently in America and academic forums in America they do capitalize it, so what the heck. lol 
https://en.wikipedia.org/wiki/Capitalization_of_Internet 
https://grammarist.com/style/internet-capitalization/

**Review by ctrlaltdavid — APPROVED** [review]

File: docs/source/host/quick-start.rst
Comment (JulianGro) [inline docs/source/host/quick-start.rst:24]:
```suggestion
1. Download the Vircadia Installer.
```
I don't see an Interface only installer on the website; Only the launcher.

File: docs/source/host/quick-start.rst
Comment (JulianGro) [inline docs/source/host/quick-start.rst:25]: Has this not been renamed to Vircadia Server yet? (I am not on Windows so I wouldn't know)

File: docs/source/host/quick-start.rst
Comment (JulianGro) [inline docs/source/host/quick-start.rst:31]:
```suggestion
5. In your Vircadia Interface, open the `Explore` app and enter `localhost` (or the server's address) to get there.
```
Same as above: You can also use hostnames like vircadia.com or localhost.

**Review by JulianGro — CHANGES_REQUESTED** [review]

**Comment by digisomni** [comment]
I need to update the image to the latest (showing Vircadia Server).

**Comment by digisomni** [comment]
Will wait for PR #138 to be merged first before addressing and merging this.

**Comment by JulianGro** [comment]
Could you rebase this on master?

**Pull request closed without merging**
